rust/noosphere-core/src/view/mutation.rs

```diff
@@ -191,6 +191,58 @@ impl SphereMutation {
             && self.delegations.changes.len() == 0
             && self.revocations.changes.len() == 0
     }
+
+    /// Consume a [SphereMutation], appending its changes to this one
+    pub fn append(&mut self, other: SphereMutation) {
+        self.content.changes = append_changes(
+            std::mem::take(&mut self.content.changes),
```

Comment:
The destination arg and return value are the same object, do we need to `take` and reassign `self`'s changes?

```rs
append_changes(&mut self.content.changes, other.content.changes);
```